pkgs/development/libraries/libspiro/default.nix

```diff
@@ -0,0 +1,19 @@
+{stdenv, autoreconfHook, autoconf, pkgconfig, fetchgit}:
+
+stdenv.mkDerivation rec {
+  name = "libspiro";
+  version = "20170128";
+  src = fetchgit {
```

Comment:
Does this rely on submodules? If not, source archives are strongly preferred. You can use `fetchFromGitHub` for convenience.